src/commands/staffama.ts

```diff
@@ -31,10 +31,10 @@ export const execute = async (
   const panelEmbed = new EmbedBuilder()
     .setTitle(`<:topgg_ico_microphone:1026877531296649256> Top.gg Public AMA`)
     .setDescription(
-      `Ask your question(s) for our upcoming **Public AMA**, hosted <t:1757163600:F> in **[Stage](https://discord.com/channels/264445053596991498/829020599200383016)**!\n\n` +
+      `Ask your question(s) for our upcoming **Public AMA**, hosted <t:1765044000:f> in **[Stage](https://discord.com/channels/264445053596991498/829020599200383016)**!\n\n` +
         `Ever wanted to know what we have on our roadmap, how Top.gg is doing, or have personal questions for our Team? Now is your chance to ask them! ${emoji.pog}\n\n` +
-        `Submit your questions by <t:1757163600:F>. Please note that we can't answer all questions!\n\n` +
-        `> Set your reminder for our AMA here: https://discord.gg/dbl?event=1411780560442884337\n\n` +
+        `Submit your questions by <t:1765044000:f>. Please note that we can't answer all questions!\n\n` +
```

Comment:
Very nitpicky but I think it'd be nice to instead define a Date instance and extract the timestamp from that to make it clearer what the date represents